metric_learn/base_metric.py

```diff
@@ -85,6 +94,24 @@ def _prepare_inputs(self, X, y=None, type_of_inputs='classic',
                        tuple_size=getattr(self, '_tuple_size', None),
                        **kwargs)
 
+  @abstractmethod
+  def get_metric(self):
+    """Returns a function that returns the learned metric between two points.
+    This function will be independent from the metric learner that learned it
+    (it will not be modified if the initial metric learner is modified).
```

Comment:
maybe add that the returned function can be directly plugged into the `metric` argument of sklearn's estimators

Reply:
Agreed, will do